SofascoreAcademy/SofascoreAcademy/Public/KeyChain.swift

```diff
@@ -0,0 +1,65 @@
+import Security
+import Foundation
+
+public enum Keychain {
+    
+    static func saveTokenToKeychain(token: String) {
+        guard let data = token.data(using: .utf8) else {
+            return
+        }
+        
+        let query: [String: Any] = [
+            kSecClass as String: kSecClassGenericPassword,
+            kSecAttrAccount as String: "academy_token",
```

Comment:
ja bi ovaj key academy_token spremila kao static varijablu 
na taj nacin ako ce se to mijenjat mijenjas samo na jednom mjestu plus smanjujes si vjerojatnost tipfelera